packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -90,8 +90,9 @@ const followProgress = (jobId, client, output) => {
 
       spinner.text = `Copy in progress: ${currentProgress}%`
     },
-    error: (err) => {
-      spinner.fail(`There was an error copying the dataset: ${err.message}`)
+    error: (event) => {
+      spinner.fail()
+      throw new Error(`There was an error copying dataset with job id: ${event.data.job_id}`)
```

Comment:
This will return undefined if the Job Id does not exist. Can we just output event.data? That will output all the info.

![image](https://user-images.githubusercontent.com/1180172/172192861-79f78b38-954d-4b5e-9eab-2128801f4280.png)

![image](https://user-images.githubusercontent.com/1180172/172192911-ed025e91-3818-48a8-bc59-5d5e24ba87aa.png)

Reply:
huh I didn't know it's possible to not have a job id